tests/lib/forms-schema.test.js

```diff
@@ -4657,7 +4657,7 @@ describe('submission event conditional logic', () => {
     )
     expect(result.error).toBe(null)
   })
-  test('should reject element with missing property in between conditional logic', () => {
+  test.only('should reject element with missing property in between conditional logic', () => {
```

Comment:
Need to remove this.

Can we please also write a test that confirms the changes are working :)